### ISSUE TITLE
Can't flash hyperos dialer zip completely in magisk

### ISSUE THREAD

**Issue by Rajpoot9002** [Mods-Center/HyperOS-Dialer#9]
Hyperos dialer not completely flash in magisk 28.1
Volume +- option not showing and reboot icon popup.
For better understanding screen shot attached.
Please resolve error.
![IMG_20241222_095011](https://github.com/user-attachments/assets/8a6157a0-87c3-4ef9-9d06-f8ac7410071f)

**Comment by ZetaMap** [comment]
First bug, the logs are not shown. And after seeing it in logcat, it's trying to unzip extra things from META-INF/extra. I will try to remove the code that unzip this thing, and if it's working, I make you a tutorial

**Comment by ZetaMap** [comment]
Yeah, it's trying to unzip META-INF/add-ons/extras.zip, but it's not present.

**Comment by ZetaMap** [comment]
mmmm yea, not working, i tried to fit it and others bugs come.... The way i do, is to extract a specific folder for my android version, and repack a new module with a generic installer. And it's working....mm a bit...

**Comment by Rajpoot9002** [comment]
So what's next? Is there any other dialer module like hyperos ?

**Comment by ZetaMap** [comment]
for me i repacked the module with the files that needed to be added.
I have a Redmi 12 5g, so for you it's probably different, but you can try.
There is the repacked module: https://www.mediafire.com/file/65r96mhibtgtq8t/kashis_HyperOS-Dialer.zip/file

Download it quickly because i don't have an account, so the file will be deleted after 14 days

**Comment by ZetaMap** [comment]
@Rajpoot9002 there are a new version is the releases section. i not tested yet but it's installing correctly, so try to use it.
